### ISSUE TITLE
Unable to control fans on Nuvoton NCT6687D

### ISSUE THREAD

**Issue by hardpoorcorn** [Rem0o/FanControl.Releases#2768]
I am attempting to control PWM fans connected to motherboard headers on an MSI MAG X670E Tomahawk Wifi motherboard.  This motherboard uses a Nuvoton NCT6687D Chip for sensors.  While Fan Control reads the proper RPMs reported by the sensor, I cannot get those fans to respond.  I have tried manual control, force settings, and paired the sensors manually, but am still unable to control the speeds via curve for flat settings.

There are no errors reported in software logs.  Is there anything else you might suggest I try?  And if not, is there any other information I can provide to help troubleshoot the issue?

**Comment by Rem0o** [comment]
Go into your BIOS and see how your headers are configured.   PWM/DC, manual mode, auto and so on.  If PWM doesn't work, try DC.  Set a manual speed like 50% in there and remove any "auto" feature in the bios.

**Comment by hardpoorcorn** [comment]
> Go into your BIOS and see how your headers are configured. PWM/DC, manual mode, auto and so on. If PWM doesn't work, try DC. Set a manual speed like 50% in there and remove any "auto" feature in the bios.

Thank you for your quick reply.  The fans are PWM, plugged into PWM headers.  Smart Fan control is disabled.  I notcied that this software relies upon librehwmonitor for the interface to the controller.  I downloaded the latest version of that software, And was similarly unable to get the fans to respond to control commands.  As such I will file an issue there, and close this here, as I believe this to be unrelated to fancontrol.
